client/packages/system/src/Item/ListView/ListView.tsx

```diff
@@ -9,10 +9,15 @@ import {
   useTranslation,
   useUrlQueryParams,
   ColumnAlign,
-  useFormatNumber,
+  NumUtils,
 } from '@openmsupply-client/common';
-import { useItems, ItemRowFragment } from '../api';
+import { useItems, ItemsWithStatsFragment } from '../api';
```

Comment:
I added a fragment to reflect the query result, this is still a bit loose in terms of type safety, I think ideally UI will be tightly tied to the data that spawns it, the query (but that would mean removing context).

Sorry this is a little bit out of scope